nbs/docs/how-to-guides/MLFlow.ipynb

```diff
@@ -0,0 +1,672 @@
+{
```

Comment:
<p>Please comment this so that it doesn't run when running the tests locally. I recently had to recreate an environment because of a cell like this.</p><pre class="ql-syntax" spellcheck="false">%%capture
# !pip install mlflow mlflavors
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>